tests/testthat/test-hclust-param.R

```diff
@@ -61,3 +61,14 @@ test_that("clusterRows works with the dynamic tree cut", {
     expect_identical(length(out), nrow(m))
 })
 
+test_that("dist.fun parameter works correctly", {
+  m <- matrix(runif(1000), ncol=10)
+  dist_result <- clusterRows(m, HclustParam(metric = "euclidean"))
```

Comment:
You should add here a condition "this test is only run when vegan package is installed"